catalyst-gateway/tests/schemathesis_tests/Earthfile

```diff
@@ -8,13 +8,19 @@ package-schemathesis:
 
     RUN apk add --no-cache gcc musl-dev
     RUN python -m pip install schemathesis==$version
+    RUN python -m pip install cbor2
     RUN mkdir /results
     COPY ./hooks/hooks.py .
     VOLUME /results
 
     ENTRYPOINT export SCHEMATHESIS_HOOKS=hooks \
                 && st run \
-                    --exclude-path-regex 'draft' \
+                    # remove these includes when endpoints will be not "draft"
```

Comment:
Why including draft endpoints here? they already get tested in nighly

Reply:
An intention of this PR was enabling `schemathesis` for every CI run, the same as we have for non draft.
Right now, these endpoints are kind of stable, so they should be identified from the Schemathesis perspective that they are released.